src/GovUk.Education.ExploreEducationStatistics.Admin/Services/BoundaryLevelService.cs

```diff
@@ -0,0 +1,66 @@
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Statistics;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Data.Model.Repository.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services;
+
+public class BoundaryLevelService : IBoundaryLevelService
+{
+    private readonly IBoundaryLevelRepository _boundaryLevelRepository;
+
+    public BoundaryLevelService(
+        IBoundaryLevelRepository boundaryLevelRepository)
+    {
+        _boundaryLevelRepository = boundaryLevelRepository;
+    }
```

Comment:
You could neaten this up by using a primary constructor so this is replaced by

```csharp
public class BoundaryLevelService(IBoundaryLevelRepository boundaryLevelRepository) : IBoundaryLevelService
```

I would also be tempted to not use a repository and depend on `StatisticsDbContext` directly for these non-complex CRUD operations.

Reply:
I've implemented the primary constructor, but this class existed before and made use of a separate repo layer, so I'd rather maintain that pattern than bring in database-related dependencies and muddying the current separation.